pkg/scheduler/apis/config/scheme/scheme_test.go

```diff
@@ -424,6 +425,7 @@ leaderElection:
   resourceName: ""
   resourceNamespace: ""
   retryPeriod: 0s
+parallelism: 8
```

Comment:
Sorry for the back and forth, but sometimes my attention is caught by other files.

This test is named "TestCodecsEncodePluginConfig" so we shouldn't be testing Parallelism here.
But rather than adding another test case, just remove the change in this file. We already have a test in defaults_test.go

The tests in this file are really about plugins, because they have a complex setup in the serializer infrastructure. The rest of defaults should be tested where they are (defaults.go)

Reply:
The tests fail if I don't make this change. The default value propagated was 16. I swapped the value to 8 to test with a non default value

Reply:
true. Just leave it in 16 then (that is, remove the other line 351)

Reply:
I have updated this as per recommendation. I thought we were planning to test the non default parallelism here as well

Reply:
No, for the reasons above

/hold cancel